pkg/oc/cli/set/env.go

```diff
@@ -371,21 +371,32 @@ func (o *EnvOptions) RunEnv() error {
 							continue
 						}
 
+						// TODO: For Maciej: the ConvertInteralPodSpecToExternal should not be needed?
```

Comment:
I am removing the need for this in https://github.com/openshift/origin/pull/20645/files#diff-54f6ccf572703b420951c9249df04b84L522